Website/src/rendering/src/components/Speakers/SpeakersGrid.tsx

```diff
@@ -20,22 +18,27 @@ export type SpeakersGridProps = ComponentProps & {
 const SpeakersGrid = (props: SpeakersGridProps): JSX.Element => {
   const speakers =
     props.fields.data?.item?.children?.results &&
-    props.fields.data.item.children.results.map((speaker, index) => (
-      <Link key={index} href={`/speakers/${speaker.itemName}`} passHref>
-        <a className="section__speakers__grid__speaker">
-          <Image className="speaker__image" field={speaker.picture.jsonValue} alt={speaker.name} />
-          <Text className="speaker__name" tag="p" field={speaker.name} />
-          <Text className="speaker__role" tag="p" field={speaker.role} />
-          {speaker.featured?.value && (
-            <div className="speaker__featured" title="Featured">
-              <FontAwesomeIcon className="icon" icon={faStar} />
-            </div>
-          )}
-        </a>
-      </Link>
-    ));
+    props.fields.data.item.children.results
+      .filter((item) => !item.featured.value)
+      .map((speaker, index) => (
+        <Link key={index} href={`/speakers/${speaker.itemName}`} passHref>
+          <a className="section__speakers__grid__speaker">
+            <Image
+              className="speaker__image"
+              field={speaker.picture.jsonValue}
+              alt={speaker.name.value}
+            />
+            <Text className="speaker__name" tag="p" field={speaker.name} />
+            <Text className="speaker__role" tag="p" field={speaker.jobTitle} />
+          </a>
+        </Link>
+      ));
 
-  return <div className="section__speakers__grid container">{speakers}</div>;
+  return (
+    <div className="section__speakers container">
+      <div className="section__speakers__grid">{speakers}</div>
```

Comment:
returned HTML can be reset to its previous value. Same with the CSS that can be simplified back to the structure it had before changes.